solana-programs/payment-router/README.md

```diff
@@ -0,0 +1,46 @@
+# Payment Router
+
+This program transfer SOL AUDIO or SOL USDC from its PDA token account to given recipients.
+
+The program has a PDA (Program Derived Address) account that will own the tokens.
+The program methods are permissionless, meaning that anyone can interact with them as long as they're willing to pay for the transactions.
```

Comment:
it's permissionless? what does that mean in this context?

Reply:
hmm yeah maybe that doesnt make sense here. i'll remove that word, although it is permissionless. anyone can create the pda and use the transfer method